packages/contracts-bedrock/test/SystemConfig.t.sol

```diff
@@ -15,83 +15,53 @@ import { Proxy } from "src/universal/Proxy.sol";
+        owner = cfg.finalSystemOwner();
+        overhead = cfg.gasPriceOracleOverhead();
+        scalar = cfg.gasPriceOracleScalar();
+        batcherHash = bytes32(uint256(uint160(cfg.batchSenderAddress())));
+        gasLimit = uint64(cfg.l2GenesisBlockGasLimit());
+        unsafeBlockSigner = cfg.p2pSequencerAddress();
+        systemConfigImpl = mustGetAddress("SystemConfig");
+        optimismMintableERC20Factory = mustGetAddress("OptimismMintableERC20FactoryProxy");
     }
-}
 
-contract SystemConfig_Initialize_Test is SystemConfig_Init {
     /// @dev Tests that initailization sets the correct values.
     function test_initialize_values_succeeds() external {
-        assertEq(sysConf.l1CrossDomainMessenger(), l1CrossDomainMessenger);
-        assertEq(sysConf.l1ERC721Bridge(), l1ERC721Bridge);
-        assertEq(sysConf.l1StandardBridge(), l1StandardBridge);
-        assertEq(sysConf.l2OutputOracle(), l2OutputOracle);
-        assertEq(sysConf.optimismPortal(), optimismPortal);
-        assertEq(sysConf.optimismMintableERC20Factory(), optimismMintableERC20Factory);
-        assertEq(sysConf.batchInbox(), batchInbox);
-        assertEq(sysConf.owner(), alice);
-        assertEq(sysConf.overhead(), overhead);
-        assertEq(sysConf.scalar(), scalar);
-        assertEq(sysConf.batcherHash(), batcherHash);
-        assertEq(sysConf.gasLimit(), gasLimit);
-        assertEq(sysConf.unsafeBlockSigner(), unsafeBlockSigner);
+        assertEq(systemConfig.l1CrossDomainMessenger(), address(l1CrossDomainMessenger));
+        assertEq(systemConfig.l1ERC721Bridge(), address(l1ERC721Bridge));
+        assertEq(systemConfig.l1StandardBridge(), address(l1StandardBridge));
+        assertEq(systemConfig.l2OutputOracle(), address(l2OutputOracle));
+        assertEq(systemConfig.optimismPortal(), address(optimismPortal));
+        assertEq(systemConfig.optimismMintableERC20Factory(), optimismMintableERC20Factory);
+        assertEq(systemConfig.batchInbox(), batchInbox);
+        assertEq(systemConfig.owner(), owner);
+        assertEq(systemConfig.overhead(), overhead);
+        assertEq(systemConfig.scalar(), scalar);
+        assertEq(systemConfig.batcherHash(), batcherHash);
+        assertEq(systemConfig.gasLimit(), gasLimit);
+        assertEq(systemConfig.unsafeBlockSigner(), unsafeBlockSigner);
         // Depends on start block being set to 0 in `initialize`
-        assertEq(sysConf.startBlock(), block.number);
+        assertEq(systemConfig.startBlock(), block.number);
         // Depends on `initialize` being called with defaults
         ResourceMetering.ResourceConfig memory cfg = Constants.DEFAULT_RESOURCE_CONFIG();
-        ResourceMetering.ResourceConfig memory actual = sysConf.resourceConfig();
+        ResourceMetering.ResourceConfig memory actual = systemConfig.resourceConfig();
         assertEq(actual.maxResourceLimit, cfg.maxResourceLimit);
         assertEq(actual.elasticityMultiplier, cfg.elasticityMultiplier);
```

Comment:
This `cfg` var now shadows the state var. Should it use the state var, or is the shadowing ok? If the latter, let's add a comment clarifying that or use a different var name

Reply:
Good catch!